package.json

```
@@ -107,6 +107,7 @@
     "@eslint-community/eslint-plugin-eslint-comments": "^4.5.0",
     "@eslint/compat": "^1.3.2",
     "@eslint/js": "^9.35.0",
+    "@playwright/experimental-ct-react": "1.56.1",
     "eslint-plugin-react-perf": "^3.3.3",
```

Biome (2.1.2)

[error] The key eslint-plugin-react-perf was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)